apps/api/scripts/initialize-db.ts

```diff
@@ -79,20 +79,20 @@ export async function initializeDatabase(): Promise<void> {
     const sqlSections = splitSqlIntoSections(sql);
 
     // Execute each section separately to better isolate errors
-    for (let i = 0; i < sqlSections.length; i++) {
+    for (const [i, sqlSection] of sqlSections.entries()) {
```

Comment:
Was using `forEach` before and that was causing problems because `forEach` doesn't wait for the callback to complete when the callback is async. So all the tables were being created in parallel. This is a problem when defining foreign keys etc. This is now synchronous.

Reply:
ah - good check